app/explore/_features/browse-projects/browse-projects.tsx

```diff
@@ -0,0 +1,137 @@
+"use client";
+
+import { ReactNode, useCallback, useMemo } from "react";
+
+import { Section } from "@/app/explore/_components/section/section";
+import {
+  BrowseProjectsContextProvider,
+  useBrowseProjectsContext,
+} from "@/app/explore/_features/browse-projects-filters/browse-projects-filters.context";
+
+import { ProjectReactQueryAdapter } from "@/core/application/react-query-adapter/project";
+
+import {
+  CardProjectMarketplace,
+  CardProjectMarketplaceLoading,
+} from "@/design-system/molecules/cards/card-project-marketplace";
+import { Tabs } from "@/design-system/molecules/tabs/tabs";
+
+import { EmptyStateLite } from "@/shared/components/empty-state-lite/empty-state-lite";
+import { ErrorState } from "@/shared/components/error-state/error-state";
+import { ShowMore } from "@/shared/components/show-more/show-more";
+import { PROJECT_TAG, PROJECT_TAG_METADATA } from "@/shared/constants/project-tags";
+import { Translate } from "@/shared/translation/components/translate/translate";
+
+import { BrowseProjectsFilters } from "../browse-projects-filters/browse-projects-filters";
+
+const tabs: { id: PROJECT_TAG | "ALL"; children: ReactNode }[] = Object.values(PROJECT_TAG).map(tag => ({
+  id: tag,
+  children: <Translate token={PROJECT_TAG_METADATA[tag].label} />,
+}));
+
+const ALL_TAB = {
+  id: "ALL",
+  children: <Translate token={"common:all"} />,
+} as const;
+
+tabs.unshift(ALL_TAB);
```

Comment:
why this is for ?

Reply:
'ALL' isn't a real tag so I had to add it